frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.test.tsx

```diff
@@ -169,10 +169,14 @@ describe("Learning Resource Expanded", () => {
         .getByRole("heading", { name: "Info" })!
         .closest("section")!
 
-      const price = run.prices?.[0]
+      const price = run.resource_prices?.[0]
 
       const displayPrice =
-        parseFloat(price!) === 0 ? "Free" : price ? `$${price}` : null
+        parseFloat(price.amount!) === 0
```

Comment:
Mentioning: We shouldn't have been using `!` here in the first place, and now it has no effect.

`!` is Typescript's [non-null assertion](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-), but `price.amount` has type `string` (already non-null) so it doesn't do anything.

We shouldn't really use it because it can lead to errors (it's silencing a type error that could be real). We have ended up using it in a few places because DRF Spectacular's types are sometimes nullable where they shouldn't be. (At least, I'm pretty sure DRF spec makes them nullable in some places it shouldn't be. Perhaps there's some reason I don't understand that it is technically correct.)